synapse/tests/test_lib_remcycle.py

```diff
@@ -0,0 +1,624 @@
+import synapse.lib.remcycle as s_remcycle
+import synapse.models.inet as s_inet
+
+from synapse.tests.common import *
+
+from tornado.testing import AsyncTestCase
+
+
+def get_vertex_global_config():
+    gconfig = {
+        'apis': {
+            'http': {
+                'doc': 'Get the vertex project landing page.',
+                'url': 'http://vertex.link/',
+                'http': {'validate_cert': False}
+            },
+            'https': {
+                'doc': 'Get the vertex project landing page.',
+                'url': 'https://vertex.link/',
+                'http': {'validate_cert': False}
+            }
+        },
+        'doc': 'Grab Vertex.link stuff',
+        'http': {
+            'user_agent': 'ClownBrowser'
+        },
+        'namespace': 'vertexproject',
+    }
+    return gconfig
+
+
+def get_bad_vertex_global_config():
+    gconfig = {
+        'apis': {
+            'fake_endpoint': {
+                'doc': 'Fake endpoint',
+                'url': 'https://vertex.link/foo/bar/duck',
+                'http': {'validate_cert': False}
+            }
+        },
+        'doc': 'Grab Vertex.link stuff',
+        'http': {
+            'user_agent': 'ClownBrowser'
+        },
+        'namespace': 'vertexproject',
+    }
+    return gconfig
+
+
+def get_ipify_global_config():
+    gconfig = {
+        'apis': {
+            'jsonip': {
+                'doc': 'Get IP in a JSON blob',
+                'http': {
+                    'validate_cert': False
+                },
+                'url': 'https://api.ipify.org?format=json',
+            }
+        },
+        'doc': 'API for getting calling IP address',
+        'http': {
+            'user_agent': 'SynapseTest'
+        },
+        'namespace': 'ipify',
+    }
+    return gconfig
+
```

Comment:
As a note - this will be a documented pep8 deviation (using 1 line between top level functions or classes instead of two).